### PR TITLE
Add static tombstone method support

### PR DESCRIPTION
**Pull request by lyudmild**

### CODE REVIEW

**Comment by scheb** [comment]
Sorry for the delayed builds. I have to push a button for each build, GitHub obviously has introduced that restriction to battle crypo miners.

**Comment by lyudmild** [comment]
Sorry for all the commits. I haven't worked with php-cs-fixer before.

File: src/analyzer/Stock/TombstoneNodeVisitor.php [hunk 1]
```diff
@@ -131,6 +147,23 @@ private function isTombstoneFunction(FuncCall $node): bool
...
+        }
+
+        $fqn = (string) $node->class.'::'.(string) $node->name;
+
+        // Unambiguous calls resolving to a FQN
+        if (\in_array($fqn, $this->tombstoneFunctionNames)) {
+            return true;
+        }
+
+        return false;
```
Comment (scheb) [inline src/analyzer/Stock/TombstoneNodeVisitor.php]: This can be a simple `return \in_array($fqn, $this->tombstoneFunctionNames);`

**Review by scheb — CHANGES_REQUESTED** [review]
Looks good. Would you please simply that one statement that I commented, then we'd be ready to merge.

File: src/analyzer/Stock/TombstoneNodeVisitor.php (hunk 1 above)
Comment (lyudmild) [reply src/analyzer/Stock/TombstoneNodeVisitor.php]: I have addressed that. Thank you!

**Comment by scheb** [comment]
Thank you for your contribution!

Released as v1.4.0

**Pull request merged**